Example/Tests/SupportedFormatsViewControllerTests.swift

```diff
@@ -15,29 +15,29 @@ final class SupportedFormatsViewControllerTests: XCTestCase {
     let initialGiniConfiguration = GiniConfiguration.shared
     
     var sections: [SupportedFormatCollectionSection] = [
-        (NSLocalizedString("ginivision.supportedFormats.section.1.title",
```

Comment:
I don't understand why you don't use the same `.localized` method thing.
If it's because you also want to check is the proper, localized string is used, why not just used the actual string people would see at runtime?

Reply:
You are right, makes sense to use the same method for the tests.